### PR TITLE
[Bugfix] Layout Fixes

### PR DESCRIPTION
**Pull request by charlotteds**
## Related Links

- Task Link: https://www.notion.so/Layout-fixes-d657a55e93bc4182b592dfc3cd091271
- Route Link: /

## Description
- Spacing between testimonial and footer
- Rounded edges for bottom of chatbox form
- Added paw trails

## Notes
- Fixed content for second testimonial.

## Screenshots
**Fixed Chatbox bottom border**
image

**Space added below testimonials section**
Screenshot 2022-04-28 at 2 44 28 PM

### CODE REVIEW

**Review by ume45876139 — APPROVED** [review]
# Working

<details>
<summary>PC</summary>

スクリーンショット 2022-04-29 10 56 52

https://user-images.githubusercontent.com/39412730/165874404-af328b0b-a60c-4c40-be82-7388aae1e4f1.mov

</details>

<details>
<summary>Tablet</summary>

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-29 at 10 57 11

</details>

<details>
<summary>Mobile</summary>

https://user-images.githubusercontent.com/39412730/165874484-aa70a73e-79c9-47ca-af02-42cf249e8b12.MP4

</details>

- Chat box deformation is fixed
- Paw trail is displayed

# Testing data
test device: Chrome, Safari
test time: 0.25h
comment: 0
iteration: 0
total test time: 0.25h

**Comment by CaneteJoash** [comment]
### Comment 
- Please fix the margin of the content in testimonials. refer my screenshots below. 
image

- Please fix the highlighted color in the content. refer my screenshots below.
image

**Comment by ume45876139** [comment]
# Working
## Chrome
<details>
<summary>PC</summary>

スクリーンショット 2022-04-29 15 04 28

</details>

- Testimonial section is fixed

# Testing data
test device: Chrome, Safari
test time: 0.1h
comment: 0
iteration: 0
total test time: 0.35h

File: src/pages/components/feature/FeatureCard.js [hunk 1]
```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 
-const FeatureCard = ({ item }) => {
+const FeatureCard = ({ item, index, length }) => {
+  const displayPaws = () => {
+    if (index < length - 1) {
+      // ? '-bottom-32 lg:-bottom-48 left-[200px] lg:left-[300px]'
+      // : 'left-[200px] lg:left-[300px]'
```
Comment (CharlotteDS) [inline src/pages/components/feature/FeatureCard.js:7]: Delete comments if unused.

**Comment by CaneteJoash** [comment]
### Approved #9 
- able to fixed the margin of the content in testimonials. refer my screenshots below.
- Please fix the highlighted color in the content. refer my screenshots below.

### Screenshots
image

test time: 0.1h
comment: 1
iteration: 0
total test time: 0.35h

**Pull request merged**